lib/pbench/agent/tool_meister.py

```diff
@@ -177,6 +180,45 @@ def start(self):
 
             args = [f"python2 {script_path}"]
             self.process = subprocess.Popen(args, shell=True)
+        elif self.name == "pcp":
+            self.logger.debug("PMCD STARTUP")
+            try:
+                pcp_reg = PbenchAgentConfig(os.environ["_PBENCH_AGENT_CONFIG"]).pmcd_reg
+            except Exception as exc:
+                self.logger.error(
+                    "Unexpected error encountered logging pbench agent configuration: '%s'",
+                    exc,
+                )
+                self.failure = True
+                return 0
+
+            with open("pcp-meister.log", "w") as pcp_logs:
+                args = ["podman", "pull", pcp_reg]
+                try:
+                    pcp_pull = subprocess.Popen(args, stdout=pcp_logs, stderr=pcp_logs)
+                    pcp_pull.wait()
```

Comment:
I'm seeing this a lot; I think we need a Podman control module somewhere to encapsulate this and avoid inlining Podman interactions all over the place.

In fact, there _is_ apparently a Python3 `podman` package; I wonder how much that would help us?

Reply:
The python package requires the user to have the io.podman.socket systemd unit file enabled (not done by default). All the podman calls are a bit repetitive though, can probably be tidied up.

Reply:
I created a new podman module, but leaving it out of this PR (will add in new PR if it still makes sense after PR #2005)